orca-web/src/main/groovy/com/netflix/spinnaker/orca/data/jackson/StageMixins.groovy

```diff
@@ -1,25 +0,0 @@
-/*
- * Copyright 2014 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.orca.data.jackson
-
-import com.fasterxml.jackson.annotation.JsonIgnore
-import com.netflix.spinnaker.orca.pipeline.model.Pipeline
-
-interface StageMixins {
-  @JsonIgnore
-  Pipeline getPipeline()
-}
```

Comment:
this would make it so any PipelineStage rendered by the object mapper wouldn't render the back ref to its owning pipeline (if there was a getPipeline on PipelineStage ever)

should make sure that a before and after of this change emit essentially the same JSON